src/constants.ts

```diff
@@ -7,6 +7,9 @@ export const jsx: ParserPlugin = 'jsx';
 
 export const newLineCharacters = '\n\n';
 
+export const ChunkSideEffectNode = 'side-effect-node';
+export const ChunkSideOtherNode = 'other-node';
```

Comment:
Same here 
```suggestion
export const chunkSideOtherNode = 'other-node';
```

Reply:
Ah yeah you're right, I usually try to follow existing conventions, but I missed that.